cpp/ql/lib/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.3.0
+
+### Deprecated APIs
+
+* The `BarrierGuard` class has been deprecated. Such barriers and sanitizers can now instead be created using the new `BarrierGuard` parameterized module.
+
+### New Features
+
+* An `isBraced` predicate was added to the `Initializer` class which holds when a C++ braced initializer was used in the initialization.
```

Comment:
Actually I don't quite understand what happened. Whatever the cause is, I don't think it's for lack of a merge-back. The file it was derived from was supposed to have been deleted by the large merge commit, but it's still here 😕

Reply:
The same change note file was independently merged on main and the old release branch, and the version on main didn't get deleted and so it showed up here again. I've manually deleted the text.

Since this PR deletes the change note file again, it shouldn't show up anymore.